examples/runner-certificates/README.md

```diff
@@ -0,0 +1,216 @@
+# Example - Spot Runner - Private subnet
+
+In this scenario the runner agent is running on a single EC2 node.
+
+The example is intended to show how the runner can be configured for self-hosted Gitlab environments with certificates signed by a custom CA.
+
```

Comment:
suggestion: as you wrote in the PR, this is not usable for the `docker+machine` executor. Could you please add a note?